content/docs/_index.md

```diff
@@ -5,6 +5,12 @@ weight: 20
 menu:
   main:
     weight: 20
+
+cascade:
+  github_repo: https://github.com/tinygo-org/tinygo-site
+  github_subdir: content/docs
+  path_base_for_github_subdir: content/docs
+  github_branch: release
```

Comment:
Shouldn't this be the `dev` branch? (To avoid having to mess with the git history on the next update).

Reply:
ok yes. good point. corrected.